.prettierrc

```diff
@@ -0,0 +1,8 @@
+printWidth: 120
+tabWidth: 2 # Default
+semi: false
+singleQuote: true
+quoteProps: as-needed # Default
+trailingComma: all
+bracketSpacing: false
+arrowParens: always # Default
```

Comment:
@gvidon Have we decided to change this rule?
https://prettier.io/docs/en/options.html#arrow-function-parentheses
But I agree that life would be easier with brackets. :)